packages/account/src/Components/poi/idv-document-submit/idv-document-submit.jsx

```diff
@@ -79,6 +80,8 @@ const IdvDocumentSubmit = ({ handleBack, handleViewComplete, selected_country, i
     const validateFields = values => {
         const errors = {};
         const { document_type, document_number } = values;
+        const sequential_number = sequentialNumberCheck(document_number).includes(true);
+        const recurring_number = recurringNumberRegex(document_number);
```

Comment:
```suggestion
        const recurring_numbers = getRecurringNumbers(document_number);
```
@hubert-deriv this variable can contain an array of matches or null, so i suggest reconsidering the way we name it.  Do I get its purpose correctly? 🙂